packages/cln_plugin/lib/src/cln_plugin_base.dart

```diff
@@ -160,6 +160,10 @@ class Plugin implements CLNPlugin {
     throw Exception("Method with name $name not found!");
   }
 
+  void _handlingNotification({required String method}) {
+    var notification = notifications[method];
```

Comment:
```suggestion
    var notification = notifications[method];
    await notification.call(this, request);
```

Ehi, don't miss yourself on something trivial.

The plugin when receiving a notification needs to make an action, otherwise, it is mindless this search inside a map without a function!